package-res/resources/web/dojo/pentaho/common/TemplatePicker.js

```diff
@@ -83,6 +83,9 @@ define(["dojo/_base/declare", "dijit/_WidgetBase", "dijit/_Templated", "dojo/on"
             },
 
             imgClick: function (event) {
+              if(event.detail > 1){
```

Comment:
Looks like this line does not match existing code style (even within this single function) - spaces after 'if' and before '{'.